### ISSUE TITLE
UX fix up - hidden views - how to find out where MUCs, missed calls and the Movim integration actually are

### ISSUE THREAD

**Issue by licaon-kter** [cmeng-git/atalk-android#151]
...as current chat view shows only 1:1

Steps:
* join MUC
* wait a bit
* hit back (arrow, gesture or titlebar)

Expected:
* 1:1 chats and MUCs

Actual:
* 1:1 chats only

2.2.5 F-Droid

**Comment by cmeng-git** [comment]
Can you provide a screenshot of the chatRoom info after you have join the room?

**Comment by licaon-kter** [comment]
![muc view](https://i.imgur.com/N0rA1iP.jpg)

**Comment by cmeng-git** [comment]
By the way, do you know of any client that allow user to turn off these messages? I would like to take a look to understand the implementation.

**Comment by licaon-kter** [comment]
Chatroominfo...oh...I thought you meant "of the chatroom"

**Comment by licaon-kter** [comment]
![this one](https://i.imgur.com/q4ooA0R.jpg)

**Comment by cmeng-git** [comment]
Can you elaborate more on what exactly the problem your are referring to?

I performed the following steps:
a. Open a 1:1 chat -> press back key
b. Open the MUC room -> press back key
c. Enter 1:1 chat again
d. Slide left will bring me to the MUC.
e. Slide right will bring me back to 1:1 room

So I am not sure what is the problem you are referring to.

**Comment by licaon-kter** [comment]
Slide right? When did you mention this to the user exactly? There are screens hidden to the right? Great...but...hidden...there's a Movim pod there too, wtf?

Let me rephrase...

**Comment by licaon-kter** [comment]
BTW, I keep reporting MUCs appearing and disappearing for a while now, yet it took you so long to mention that "hey...MUCs are elsewhere" ?
 ಠ_ಠ

**Comment by cmeng-git** [comment]
The problem lies in the fact I misunderstood the issue being raised. It would be good for future discussion, appreciate if you would provide a more precise description of the problem.

Actually you also caught me off guard. As an android device user, I assume the user would slide left, right, up and down to access to new page/feature; this is how android main menu UI behave.

**Comment by licaon-kter** [comment]
What other app has sliding views yet no hints in the UI that they exist?

**Comment by cmeng-git** [comment]
Open a Chrome on ipad, sliding left will show some hidden feature. Yet there is no indication of the feature in the UI.

I will include some description of aTalk UI design in online help.

**Comment by licaon-kter** [comment]
"Chrome on iPad"...really? :) A closed source webkit engine with a theme on a closed source OS...

"Android main menu" - I don't know what that is

**Comment by cmeng-git** [comment]
Any indication I do on aTalk will take up valuable display area. That was the reason why I did not do anything on this to conserve display resource; trusting that the users of android should have got brain washed by android OS/UI implementation, and  know how to navigate with sliding gestures.

**Comment by licaon-kter** [comment]
Xabber has an overlay on first start that shows some arrows left and right with info...

**Comment by cmeng-git** [comment]
Please see:
[aTalk UI design & Implementation](https://cmeng-git.github.io/faq.html#ui_01)
